public/locales/fi/date_time_picker.json

```
@@ -7,5 +7,5 @@
   "time_input.hours_label": "Tunnit",
   "time_input.minutes_label": "Minuutit",
   "date_input.error.default_min_date": "Valittu päivä on ennen aikaisinta sallittua päivää",
-  "date_input.error.default_max_date": "Valittu päivä on myöhäisimmän sallitun päivän jälkeen"
+  "som.har.valts.in.är.före.det.tidigast.tillåtna.datumet": "Valittu päivä on myöhäisimmän sallitun päivän jälkeen"
```

```suggestion
  "date_input.error.default_max_date": "The selected date is later than the latest allowed date"
```
Translation key is in swedish.